docs/guide/installation-procedure.md

```diff
@@ -41,25 +41,26 @@ First of all, you may want to configure your APT package manager by adding the C
 Simply execute the following lines:
 
 ```sh
-apt-add-repository 'deb http://www.cyberbotics.com/debian/ binary-amd64/'
+sudo su -
+apt-add-repository 'deb https://www.cyberbotics.com/debian/ binary-amd64/'
 apt-get update
 ```
 
 As an alternative, you can easily add the Cyberbotics repository from the `Software and Updates` application.
 In the `Other Software` tab, click on the `Add...` button and copy the following line:
 
 ```text
-deb http://www.cyberbotics.com/debian/ binary-amd64/
+deb https://www.cyberbotics.com/debian/ binary-amd64/
 ```
 
-When you will close the window, the APT packages list should be automatically updated.
+When you close the window, the APT packages list should be automatically updated.
 Otherwise you can manually execute the following command:
 
 ```sh
 apt-get update
 ```
 
-Optionally, Webots can be authentified thanks to the `Cyberbotics.asc` signature file which can be downloaded [here](http://www.cyberbotics.com/linux), using this command:
+Webots should be authenticated with the [Cyberbotics.asc](https://www.cyberbotics.com/Cyberbotics.asc) signature file which can be downloaded from the [Webots download page](https://www.cyberbotics.com/download), and installed using this command:
```

Comment:
This should probably go before the `apt-get update`, otherwise, this commands shows warnings.

Reply:
Yes, I believe it's nice to also see the page where it's embedded.